Boards - Getting Started (#5450)
mattermost/docs: source/boards/get-started-with-boards.rst

Select an existing card to open it or create a new card.

Description
-----------

Use the description to provide context around the reason for the card. Cards are visible to anyone who can access the board, so if there's sensitive information it's best not to include it here. You can add links to external documentation, as well as at-mention teammates.

Comments
--------

Comments are a helpful way to add additional information to a card, and comments are time and date stamped so they're easy to track. Comments are listed in descending order from most recent to oldest.

To add a comment, select the **Comment** field. When you're finished, select **Save**.

Properties
----------

You can create properties for your cards that suit the way you're using them. If you're planning a project, you might want to have properties such as **Due date**, **Assigned to**, and **Priority**. Properties are displayed on the card when you open it, and you can also choose to display your cards based on their properties, using filters.

Properties can be renamed and deleted.

To learn more about properties, visit XXX.

> Views

Author: - What do we really want to know about views - what's the biggest use case?
- Don't add too much complexity.

-----

View are a great way to assess your board from various angles. Some views are linear, while others are closer to Kanban style. Views with a more linear display, such as the task view, display less context but are easier to scroll through. Use a view such as **By Status** to see all the cards, presented in swimlanes, with more details, in the form of properties, displayed.

You can also choose views such as a calendar view, which gives you a view of your work overlaid on a calendar grid.

To learn more about views, visit XXX.

> Filters and sorting

Author: - What's the difference between filters and sorting? 
- How are filters defined?
- When would you filter and when would you sort?

Get Started with Mattermost Boards
==================================

> Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.

Reviewer: ```suggestion
Access Mattermost Boards by selecting **Boards** from the **Product menu** |product-list| icon.
```

> Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.

Reviewer 2: This page may need to be added to the TOC.
The icon doesn't seem to render properly on preview.

.. |product-list| image:: ../images/products_E82F.svg
  :height: 24px
  :width: 24px
  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.

Boards are used for planning projects collaboratively. You can access the list of channels and their boards from the drop-down menu in the left-hand sidebar. Boards are made up of cards, and cards can have a wide range of purposes.

> See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Channels to enable team collaboration.

Reviewer: would "Channels" here refer to the product vertical or the communication vehicle? If the latter, lowercase "c" :)

> See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Channels to enable team collaboration.

Author: ```suggestion
See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Boards to enable team collaboration.
```

Get Started with Mattermost Boards
==================================

Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.

.. |product-list| image:: ../images/products_E82F.svg
  :height: 24px
  :width: 24px
  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.

Boards are used for planning projects collaboratively. You can access the list of channels and their boards from the drop-down menu in the left-hand sidebar. Boards are made up of cards, and cards can have a wide range of purposes.

See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Channels to enable team collaboration.

Create a board
--------------

> When you first start using Boards, you need to create a board. You can select a template from the options provided or you can create an empty board. The only difference is that a template will be populated with sample cards and properties and in an empty template you start from scratch.

Reviewer 3: ```suggestion
When you first start using Boards, you need to create a board. You can select a template from the options provided or you can create an empty board. The only difference is that a template will be populated with sample cards, properties, and views, and in an empty template you start from scratch.
```

> To find a board, open Boards and select the drop-down menu in the sidebar. Each board is linked to the channel it was created in. This includes direct messages and group messages.

Reviewer 3: ```suggestion
To find a board, open Boards and select the drop-down menu in the sidebar. You can search for a workspace or browse through all existing workspaces. Each workspace is linked to the channel it was created in. This includes direct messages and group messages.
```

Reviewer 3: Note: We will need to change the terminology to "boards" instead of "workspaces" after we ship the new permissions system in v6.6.

Author: Thanks for the heads up! :)

See the `working with boards <https://docs.mattermost.com/boards/working-with-boards.html>`_ and `templates <https://docs.mattermost.com/boards/templates.html>`_ documentation for more information.

Create a card
-------------

> Cards are like sticky notes. You can arrange them in various ways, using different views. You can also give them mutiple areas of information called properties. Adding and using properties makes it easier to organize and find cards, using filters based on card properties.

Reviewer: ```suggestion
Cards are like sticky notes. You can arrange them in various ways, using different views. You can also give them multiple areas of information called properties. Adding and using properties makes it easier to organize and find cards using filters based on card properties.
```

> Once you've chosen a board template, you'll find pre-populated cards. You can use and edit those, or create new ones. Use the Description field to add details, and you can use Markdown to add formatting. You can add comments to a card in the Comments section.

Reviewer: ```suggestion
Once you've chosen a board template, you'll find pre-populated cards. You can use and edit those, or create new ones. Use the **Description** field to add details, and you can use Markdown to add formatting. You can add comments to a card in the **Comments** section.
```

Select an existing card to open it or create a new card.

See the `cards documentation <https://docs.mattermost.com/boards/work-with-cards.html>`_ for more information about working with cards.

Description
-----------

Use the description to provide context around the reason for the card. Cards are visible to anyone who can access the board, so if there's sensitive information it's best not to include it here. You can add links to external documentation, as well as at-mention teammates.

Comments
--------

> Comments are a helpful way to add additional information to a card, and comments are time and date stamped so they're easy to track. Comments are listed in descending order from most recent to oldest.

Reviewer 3: ```suggestion
Comments are a helpful way to provide feedback or ask questions related to a card, and comments are time and date stamped so they're easy to track. Comments are listed in descending order from most recent to oldest.
```

To add a comment, select the **Comment** field. When you're finished, select **Save**.

Properties
----------

You can create properties for your cards that suit the way you're using them. If you're planning a project, you might want to have properties such as **Due date**, **Assigned to**, and **Priority**. Properties are displayed on the card when you open it, and you can also choose to display your cards based on their properties, using filters.

Properties can be renamed and deleted.

See the `properties <https://docs.mattermost.com/boards/work-with-cards.html#add-and-manage-properties>`_ documentation for more information.

Views
-----

> View are a great way to assess your board from various angles and allow your team to visualize the same cards and data from different perspectives, so they can stay up-to-date in the way that works best for them. Some views are linear, while others are closer to Kanban style. Views with a more linear display, such as the task view, display less context but are easier to scroll through. 

Reviewer 3: ```suggestion
View are a great way to assess your board from various angles and allow your team to visualize the same cards and data from different perspectives, so they can stay up-to-date in the way that works best for them. Some views are linear, while others are closer to Kanban style. Views with a more linear display, such as the table view, display less context but are easier to scroll through. 
```

> To add a new view, go to Add a new view from the view drop-down, then select from any of the following views:

Reviewer: ```suggestion
To add a new view, go to **Add a new view** from the view drop-down, then select from any of the following views:
```

> - Board: Adds a Kanban board, similar to this one, that allows your team to organize cards in swimlanes grouped by any property of your choosing. This view helps you visualize your project progress.

Reviewer 3: ```suggestion
- Board: Adds a Kanban board that allows your team to organize cards in swimlanes grouped by any property of your choosing. This view helps you visualize your project progress.
```

- Table: Displays cards in a table format with rows and columns. Use this view to get an overview of all your project tasks. Easily view and compare the state of all properties across all cards without needing to open individual cards.
- Gallery: Displays cards in a gallery format, so you can manage and organize cards with image attachments.

> - Calendar: Adds a calendar view to easily visualize your cards by dates and keep track of deadlines.

Reviewer: ```suggestion
- **Calendar**: Adds a calendar view to easily visualize your cards by dates and keep track of deadlines.
```
